recipe/meta.yaml

```diff
@@ -18,11 +18,7 @@ source:
   git_url: https://github.com/pytorch/pytorch.git
   git_tag: v{{ version }}
   patches:
-    - patches/fix_dispatch_apply_auto.patch
-    # 2022/07 hmaarrfk
-    # We've patch out the bug in GCC that makes it incompatible
-    # https://github.com/conda-forge/ctng-compilers-feedstock/blob/7ea10c2526445beb9faff23efcc6bc40fc0079b5/recipe/meta.yaml#L23=
-    - patches/remove_gcc_maximum_requirements.patch
+    - nvml.patch  # [cuda_compiler_version == "11.2"]
```

Comment:
please try to follow the old organization, put this file in the `patches/` directory